cmd/avrogo/generate_test.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"github.com/actgardner/gogen-avro/v10/parser"
+	"github.com/actgardner/gogen-avro/v10/schema"
+	"testing"
+
+	avro "github.com/actgardner/gogen-avro/v10/schema"
+	qt "github.com/frankban/quicktest"
+)
+
+var eventNameQualifiedName = schema.QualifiedName{Namespace: "EventName", Name: "EventName"}
```

Comment:
Could you make all this package variables part of the `TestShouldImportAvroTypeGen` function?